ha/core/event_manager/event_manager.py

```diff
@@ -332,4 +345,16 @@ def message_type(self, component: str) -> str:
         Args:
             component (str): component name.
         """
+        key = const.EVENT_MGR_MESSAGE_TYPE_KEY.replace("<component_id>", component)
+        value = None
+        Log.debug(f"Fetching message type for {key}")
+
+        if self._confstore.key_exists(key):
+            kv = self._confstore.get(key)
```

Comment:
for loop not need as key will return only one key val pair 

is single component will have multiple message type

Reply:
There can be multiple keys with same prefix (not expected but keys can get added my mistake) and we want the exact matching key ex   "events/subscribe/sspl"  ,  "events/subscribe/sspl_test" etc
So added the for loop